hadoop-hdfs-project/hadoop-hdfs-httpfs/src/test/java/org/apache/hadoop/lib/util/TestConfigurationUtils.java

```diff
@@ -124,4 +127,22 @@ public void testVarResolutionAndSysProps() {
     assertEquals(conf.get("user.name"), "foo");
   }
 
+  @Test
+  public void testCompactFormatProperty() throws IOException {
```

Comment:
Should we add a test using a file in "resources" too?

Reply:
The `ConfigurationUtils#load` takes only InputStream as input and does not care about where the data come from. I think loading configs from file is covered by test cases in o.a.h.lib.server.TestServer.

Reply:
Having a file makes it cleaner than putting the whole string in code.
That being said, I think both should be there.

Reply:
I updated the code based on your suggestion.